index.html

```diff
@@ -1,293 +1,302 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
-  <meta charset="utf-8">
-  <title> Athaliah's Portflio </title>
-  <link rel="stylesheet" href="style.css">
-  <link rel="preconnect" href="https://fonts.googleapis.com">
-  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;1,100;1,600&display=swap"
-    rel="stylesheet">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Athaliah's Portfolio</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.0/css/all.min.css" integrity="sha512-10/jx2EXwxxWqCLX/hHth/vu2KY3jCF70dCQB8TSgNjbCVAC/8vai53GfMDrO2Emgwccf2pJqxct9ehpzG+MTw==" crossorigin="anonymous" referrerpolicy="no-referrer"/>
+
+  <link rel="stylesheet" href="./style.css" />
+		<link rel="stylesheet" href="./css/popup.css" />
 </head>
-
 <body>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Kindly ensure that you remove the extra space at the right of your page from the **Header** to the **bottom** on the mobile version to match the Figma design. 
![Screenshot from 2022-11-25 14-41-35](https://user-images.githubusercontent.com/83042742/203989657-018c087c-aed0-41f8-a379-99cbeb59479b.png)